modules/custom/az_core/az_core.module

```diff
@@ -529,3 +530,17 @@ function az_core_preprocess_container__text_format_filter_help(&$variables) {
     }
   }
 }
+
+/**
+ * Implements hook_preprocess_entity_embed_container().
+ *
+ * Prevents double-submit button ajax errors in edit pages
+ * by swapping embedded webforms with a custom preview when in collapsed mode.
+ */
+function az_core_preprocess_entity_embed_container(array &$variables) {
+  if (isset($variables["element"]["entity"]["webform"])) {
+    if (\Drupal::routeMatch()->getRouteName() === 'entity.node.edit_form') {
+      $variables["children"] = Markup::create("<hr /><p><h3>Webform Preview</h3></p><hr />");
```

Comment:
I think technically we shouldn't be using self-closing elements (XHTML) in since we're using a HTML5 doctype.

```suggestion
      $variables["children"] = Markup::create("<hr><p><h3>Webform Preview</h3></p><hr>");
```